index.html

```diff
@@ -24,8 +24,8 @@ <h1>BONSAI Sparql Endpoint</h1>
           <option value="activitytype">Activity Types</option>
           <option value="location">Locations</option>
           <option value="flowobject">Flow Objects</option>
-          <option value="inputFlowLocation">Input Flows by Location</option>
-          <option value="outputFlowLocation">Output Flows by Location</option>
+          <option value="inputFlowofActivitybyLocation">Input Flows of an Activity by Location</option>
+          <option value="outputFlowofActivitybyLocation">Output Flows by Location</option>
```

Comment:
@agneta20  should this be `Output Flows   of an Activity by Location` as the one above?